core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.spec.ts

```diff
@@ -520,8 +525,14 @@ describe('Question Suggestion Review Modal Controller', function() {
         ]
       };
 
-      $httpBackend.expect('GET', '/threadhandler/123').respond(responseDict);
-      $httpBackend.flush();
+      let spyObj = spyOn(
+        ThreadDataBackendApiService, 'fetchMessagesAsync')
+        .and.returnValue($q.resolve(responseDict));
+
+      $scope.init();
+      $rootScope.$apply();
+
+      expect(spyObj).toHaveBeenCalled();
```

Comment:
try to use `toHaveBeenCalledWith` instead of `toHaveBeenCalled`

Reply:
Done